Robinhood/Robinhood.py

```diff
@@ -308,374 +348,491 @@ def get_historical_quotes(
-            queries `quote` endpoint, dict wrapper
+            Note:
+                queries `quote` endpoint, dict wrapper
 
-        Args:
-            stock (str): stock ticker
-
-        Returns:
-            (str): previous close date
+            Args:
+                stock (str): stock ticker
 
+            Returns:
+                (str): previous close date
         """
+
         return self.get_quote_list(stock,'previous_close_date')
 
-    def adjusted_previous_close(self, stock=''):
-        """get adjusted previous closing price for a stock
 
-        Note:
-            queries `quote` endpoint, dict wrapper
+    def adjusted_previous_close(self, stock=''):
+        """
+            Get adjusted previous closing price for a stock
 
-        Args:
-            stock (str): stock ticker
+            Note:
+                queries `quote` endpoint, dict wrapper
 
-        Returns:
-            (float): adjusted previous closing price
+            Args:
+                stock (str): stock ticker
 
+            Returns:
+                (float): adjusted previous closing price
         """
+
         return self.get_quote_list(stock,'adjusted_previous_close')
 
+
     def symbol(self, stock=''):
-        """get symbol for a stock
+        """
+            Get symbol for a stock
 
-        Note:
-            queries `quote` endpoint, dict wrapper
+            Note:
+                queries `quote` endpoint, dict wrapper
 
-        Args:
-            stock (str): stock ticker
-
-        Returns:
-            (str): stock symbol
+            Args:
+                stock (str): stock ticker
 
+            Returns:
+                (str): stock symbol
         """
+
         return self.get_quote_list(stock,'symbol')
 
+
     def last_updated_at(self, stock=''):
-        """get last update datetime
+        """
+            Get last update datetime
 
-        Note:
-            queries `quote` endpoint, dict wrapper
+            Note:
+                queries `quote` endpoint, dict wrapper
 
-        Args:
-            stock (str): stock ticker
+            Args:
+                stock (str): stock ticker
 
-        Returns:
-            (str): last update datetime
+            Returns:
+                (str): last update datetime
+        """
 
+        return self.get_quote_list(stock, 'last_updated_at')
+
+    
+    def last_updated_at_datetime(self, stock=''):
         """
-        return self.get_quote_list(stock,'last_updated_at')
-        #TODO: recast to datetime object?
+            Get last updated datetime
 
-    def get_account(self):
-        """fetch account information
+            Note:
+                queries `quote` endpoint, dict wrapper
 
-        Returns:
-            (:obj:`dict`): `accounts` endpoint payload
+            Args:
+                stock (str): stock ticker
+
+            Returns:
+                (datetime): last update datetime
+
+        """
+
+        #Will be in format: 'YYYY-MM-ddTHH:mm:ss:000Z'
```

Comment:
move to `Notes` section in docstring instead?

Reply:
Good point - added